source/core/ExecutionContext.cpp

```diff
@@ -22,6 +22,12 @@ extern "C" {
 }
 #endif
 
+#if kVireoOS_emscripten
+extern "C" {
+    extern void jsExecutionContextDebugNodeSync(StringRef)
```

Comment:
A better name may be jsDebuggingContextDebugPointInterrupt since the purpose of this callback is to raise an interrupt back into the JS layer when we hit the DebugPoint.